int-tests/src/test-integration/java/io/neow3j/compiler/ECPointIntegrationTest.java

```diff
@@ -123,6 +138,15 @@ public static ByteString ecPointAsByteString(ECPoint ecPoint) {
             return ecPoint.toByteString();
         }
 
+        public static ECPoint ecPointFromString() {
+            return new ECPoint("0x03b4af8d061b6b320cce6c63bc4ec7894dce107bfc5f5ef5c68a93b4ad1e136816");
```

Comment:
Do we also support strings without the "0x" prefix?

Reply:
Yes, a `types.ECPublicKey` is initialised when compiling that string. So, the prefix does not matter. See here: https://github.com/neow3j/neow3j/blob/f45ce15d23cdfc6820a95c97b54d601147aa06d7/compiler/src/main/java/io/neow3j/compiler/converters/ObjectsConverter.java#L348-L353